### PR TITLE
Exposed sequence in function sig of AddScriptLink and AddCSSLink 

### PR DESCRIPTION
**Pull request by bschlintz**

### CODE REVIEW

File: src/customActions.js [hunk 1]
```diff
@@ -112,21 +112,21 @@ CustomActions.prototype.add = function (customAction) {
 };
 
 
-CustomActions.prototype.addScriptLink = function (name, url, scope = "Web") {
+CustomActions.prototype.addScriptLink = function (name, url, scope = "Web", sequence = 100) {
```
Comment (DroopyTersen) [inline src/customActions.js]: 4 params, with 2 of  them optional is a good hint to cutover to an options object. I think with should change the signature to .addScriptLink(name, url, opts)

File: src/customActions.js [hunk 2]
```diff
@@ -112,21 +112,21 @@ CustomActions.prototype.add = function (customAction) {
...
+		Sequence: sequence,
 		Location: "ScriptLink",
 		Scope: scope,
 		ScriptSrc: url
 	};
 	return this.add(customAction);
 };
 
-CustomActions.prototype.addCSSLink = function (name, url, scope = "Web") {
+CustomActions.prototype.addCSSLink = function (name, url, scope = "Web", sequence = 100) {
```
Comment (DroopyTersen) [inline src/customActions.js]: same thing as addScriptLink, use opts object and merge defaults with Object.assign

File: src/customActions.js [hunk 3]
```diff
@@ -112,21 +112,21 @@ CustomActions.prototype.add = function (customAction) {
...
 		Description: name,
 		Group: name,
-		Sequence: 100,
+		Sequence: sequence,
 		Location: "ScriptLink",
 		Scope: scope,
 		ScriptSrc: url
 	};
 	return this.add(customAction);
 };
```
Comment (DroopyTersen) [inline src/customActions.js]: merge the custom action with the options: customAction = Object.assign({}, customAction, opts || {});

**Comment by bschlintz** [comment]
That makes sense. Good catch! I've updated the code and tests to match the function signature with opts.

**Comment by bschlintz** [comment]
Dev was out of date from Master. Pulled from your master into my Dev and added into PR. All tests passing

**Pull request merged**